### DISCUSSION TITLE
I want to add and translate Spanish. Howto?

### DISCUSSION THREAD

**Discussion by luckkmaxx** [Mahmud0808/Iconify#236] [General]
Is there a weblate or similar project to be able to translate the app?

**Comment by Mahmud0808** [comment]
1. Fork the iconify repository.
2. Create a folder named `values-es` in `/app/src/main/res/` directory
3. Translate the `/values/strings.xml` file and put it inside newly created `values-es` folder 
4. Create a pull request, I will review before merging.
